src/python/pants/init/specs_calculator.py

```diff
@@ -125,6 +143,36 @@ def create(
                 filesystem_specs.append(FilesystemLiteralSpec(file_name))
             else:
                 address_specs.append(SingleAddress(address.spec_path, address.target_name))
+
+
+        if query_expr_strings:
+            # TODO(#7346): deprecate --owner-of and --changed-* in favor of --query versions, allow
```

Comment:
`--owner-of` is now removed.

Reply:
I'm not certain if we'll want to remove `--changed-since`. That is a very common pattern we teach people to use, and it's less typing to say `--changed-since=HEAD` than `--query='changed_since HEAD'`. Rather, we've been trying to find ways to make `--changed-since` even shorter to type.

I'm not sure; maybe the consistency would be worth it.